Documentation/operations/performance/index.rst

```diff
@@ -60,8 +60,7 @@ the Packet machines to use a `"Mixed/Hybrid"
 <https://www.packet.com/developers/docs/network/advanced/layer-2/>`_ network
 mode, where the secondary interfaces of the machines share a flat L2 network.
 While this can be done on the Packet web UI, we include appropriate Terraform
-files to automate this process.
-
+(version 0.13) files to automate this process.
```

Comment:
I'm fine having this change but you can also add this [0] into the top of the file https://github.com/cilium/cilium-perf-networking/blob/master/terraform/main.tf

[0]
```
terraform {
  required_version = ">= 0.13"
}
```

Reply:
Thanks!